<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="WKWZT9RwZdhxh2P" failures="6" tests="23" name="Smoke" time="23.681" errors="0" timestamp="2022-11-22T17:49:13 IST">
  <testcase classname="AllClasses.SmokeTest" name="@AfterMethod tearDown" time="0.0">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at AllClasses.SmokeTest.tearDown(SmokeTest.java:31)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- @AfterMethod tearDown -->
  <testcase classname="AllClasses.SmokeTest" name="@BeforeMethod setup" time="-1.669119553572E9">
    <skipped/>
  </testcase> <!-- @BeforeMethod setup -->
  <testcase classname="AllClasses.SmokeTest" name="@AfterMethod tearDown" time="-1.669119553574E9">
    <skipped/>
  </testcase> <!-- @AfterMethod tearDown -->
  <testcase classname="AllClasses.SmokeTest" name="@BeforeMethod setup" time="-1.669119553575E9">
    <skipped/>
  </testcase> <!-- @BeforeMethod setup -->
  <testcase classname="AllClasses.SmokeTest" name="@AfterMethod tearDown" time="-1.669119553577E9">
    <skipped/>
  </testcase> <!-- @AfterMethod tearDown -->
  <testcase classname="AllClasses.SmokeTest" name="@BeforeMethod setup" time="-1.669119553578E9">
    <skipped/>
  </testcase> <!-- @BeforeMethod setup -->
  <testcase classname="AllClasses.SmokeTest" name="@AfterMethod tearDown" time="-1.669119553579E9">
    <skipped/>
  </testcase> <!-- @AfterMethod tearDown -->
  <testcase classname="AllClasses.MyFirstTestNGTest2" name="createAccount" time="0.005"/>
  <testcase classname="AllClasses.MyFirstTestNGTest2" name="deleteAccount" time="0.0"/>
  <testcase classname="AllClasses.MyFirstTestNGTest2" name="login" time="0.0"/>
  <testcase classname="AllClasses.MyFirstTestNGTest2" name="logOutFromApplication" time="0.001"/>
  <testcase classname="AllClasses.MyFirstTestNGTest" name="test1" time="0.002"/>
  <testcase classname="AllClasses.MyFirstTestNGTest" name="test2" time="0.001"/>
  <testcase classname="AllClasses.MyFirstTestNGTest" name="test3" time="0.0"/>
  <testcase classname="AllClasses.AssertionDemo" name="login" time="0.003"/>
  <testcase classname="AllClasses.AssertionDemo" name="logOutFromApplication" time="0.001"/>
  <testcase classname="AllClasses.AssertionDemo" name="test1" time="0.002">
    <failure type="java.lang.AssertionError" message="expected:&amp;lt;12&amp;gt; but was:&amp;lt;13&amp;gt;">
      <![CDATA[java.lang.AssertionError: expected:<12> but was:<13>
at AllClasses.AssertionDemo.test1(AssertionDemo.java:13)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- test1 -->
  <testcase classname="AllClasses.AssertionDemo" name="test2" time="0.003">
    <failure type="java.lang.AssertionError" message="Count Mismatch expected [13] but found [12]">
      <![CDATA[java.lang.AssertionError: Count Mismatch expected [13] but found [12]
at AllClasses.AssertionDemo.test2(AssertionDemo.java:19)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- test2 -->
  <testcase classname="AllClasses.AssertionDemo" name="test3" time="0.001">
    <failure type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: 
at AllClasses.AssertionDemo.test3(AssertionDemo.java:25)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
... Removed 30 stack frames]]>
    </failure>
  </testcase> <!-- test3 -->
  <testcase classname="AllClasses.AssertionDemo" name="test4" time="0.001"/>
  <testcase classname="AllClasses.AssertionDemo" name="test5" time="0.001">
    <failure type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: 
at AllClasses.AssertionDemo.test5(AssertionDemo.java:39)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
... Removed 30 stack frames]]>
    </failure>
  </testcase> <!-- test5 -->
  <testcase classname="AllClasses.AssertionDemo" name="test6" time="0.0">
    <failure type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: 
at AllClasses.AssertionDemo.test6(AssertionDemo.java:47)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
... Removed 30 stack frames]]>
    </failure>
  </testcase> <!-- test6 -->
  <testcase classname="AllClasses.LoginTest" name="invalidUsernameandPassword" time="0.635"/>
  <testcase classname="AllClasses.LoginTest" name="noUsernameandPassword" time="0.052"/>
  <testcase classname="AllClasses.LoginTest" name="validUsernameandPassword" time="3.134"/>
  <testcase classname="AllClasses.LoginTest" name="validusernameinvalidPassword" time="0.086"/>
  <testcase classname="AllClasses.SmokeTest" name="verifyForgotPasswordLink" time="0.0">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at AllClasses.SmokeTest.verifyForgotPasswordLink(SmokeTest.java:69)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- verifyForgotPasswordLink -->
  <testcase classname="AllClasses.SmokeTest" name="verifySocialMediaAcocunt" time="0.0">
    <skipped/>
  </testcase> <!-- verifySocialMediaAcocunt -->
  <testcase classname="AllClasses.SmokeTest" name="verifyTitle" time="0.0">
    <skipped/>
  </testcase> <!-- verifyTitle -->
  <testcase classname="AllClasses.SmokeTest" name="verifyURL" time="0.0">
    <skipped/>
  </testcase> <!-- verifyURL -->
</testsuite> <!-- Smoke -->
